### PR TITLE
Method update to unanimous

### PR DESCRIPTION
**Pull request by lengrongfu**
## Related Issue

Issue: https://github.com/dragonflyoss/Dragonfly2/issues/532

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation Update (if none of the other choices apply)

## Checklist:

- [x] Code compiles correctly.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by 244372610 — APPROVED** [review]

File: cdnsystem/storedriver/local/local_driver.go [hunk 1]
```diff
@@ -317,7 +317,7 @@ func (ds *driver) Remove(raw *storedriver.Raw) error {
 }
 
 // GetAvailSpace returns the available disk space in Byte.
```
Comment (yxxhero) [inline cdnsystem/storedriver/local/local_driver.go:319]: GetAvailSpace --> GetFreeSpace

File: cdnsystem/storedriver/driver.go [hunk 2]
```diff
@@ -254,8 +254,8 @@ func (s *driverPlugin) MoveFile(src string, dst string) error {
 }
 
 // GetAvailSpace returns the available disk space in B.
```
Comment (yxxhero) [inline cdnsystem/storedriver/driver.go:256]: GetAvailSpace --> GetFreeSpace

File: cdnsystem/storedriver/driver.go [hunk 3]
```diff
@@ -68,7 +68,7 @@ type Driver interface {
 	Stat(raw *Raw) (*StorageInfo, error)
 
 	// GetAvailSpace returns the available disk space in B.
```
Comment (yxxhero) [inline cdnsystem/storedriver/driver.go:70]: GetAvailSpace --> GetFreeSpace

File: cdnsystem/storedriver/mock_driver.go [hunk 4]
```diff
@@ -79,9 +79,9 @@ func (mr *MockDriverMockRecorder) Get(arg0 interface{}) *gomock.Call {
 }
 
 // GetAvailSpace mocks base method.
```
Comment (yxxhero) [inline cdnsystem/storedriver/mock_driver.go:81]: GetAvailSpace --> GetFreeSpace

**Review by yxxhero — CHANGES_REQUESTED** [review]

File: cdnsystem/daemon/cdn/storage/storage_gc.go [hunk 5]
```diff
@@ -48,14 +48,14 @@ func NewStorageCleaner(cfg *GCConfig, driver storedriver.Driver, storageMgr Mana
...
-	freeSpace, err := cleaner.driver.GetAvailSpace()
+	freeSpace, err := cleaner.driver.GetFreeSpace()
 	if err != nil {
 		if cdnerrors.IsFileNotExist(err) {
 			err = cleaner.driver.CreateBaseDir()
 			if err != nil {
 				return nil, err
 			}
-			freeSpace, _ = cleaner.driver.GetAvailSpace()
+			freeSpace, _ = cleaner.driver.GetFreeSpace()
```
Comment (yxxhero) [inline cdnsystem/daemon/cdn/storage/storage_gc.go:58]: why ignore error?

**Review by yxxhero — APPROVED** [review]
LGTM

**Review by 244372610 — APPROVED** [review]

**Pull request merged**